### PR TITLE
Remove passivests

### PR DESCRIPTION
**Pull request by hwupathum**

### CODE REVIEW

File: apps/console/src/features/applications/components/wizard/application-create-wizard.tsx [hunk 1]
```diff
@@ -415,13 +409,7 @@ export const ApplicationCreateWizard: FunctionComponent<ApplicationCreateWizardP
...
 
         if (addProtocol) {
-            let configName = selectedTemplate.authenticationProtocol;
-
-            if (configName === SupportedAuthProtocolTypes.WS_FEDERATION) {
-                configName = "passiveSts";
-            } else if (configName === SupportedAuthProtocolTypes.WS_TRUST) {
-                configName = "wsTrust";
-            }
+            const configName = selectedTemplate.authenticationProtocol;
```
Comment (hwupathum) [inline apps/console/src/features/applications/components/wizard/application-create-wizard.tsx:412]: put the type

File: apps/console/src/features/applications/components/wizard/protocol-wizard-summary.tsx [hunk 2]
```diff
@@ -88,13 +88,7 @@ export const ProtocolWizardSummary: FunctionComponent<ProtocolWizardSummaryProps
      *  Set protocol image.
      */
     useEffect(() => {
```
Comment (hwupathum) [inline apps/console/src/features/applications/components/wizard/protocol-wizard-summary.tsx:90]: useEffect and state is not needed

File: apps/console/src/features/applications/constants/application-management.ts [hunk 3]
```diff
@@ -422,15 +422,12 @@ export class ApplicationManagementConstants {
 
     public static readonly CUSTOM_APPLICATION_SAML = "custom-application-saml";
 
-    public static readonly CUSTOM_APPLICATION_PASSIVE_STS = "custom-application-passive-sts";
-
     public static readonly CUSTOM_APPLICATION = "custom-application";
 
     public static readonly CUSTOM_APPLICATION_PROTOCOL_ORDER: Map<string, number> =
```
Comment (hwupathum) [inline apps/console/src/features/applications/constants/application-management.ts:427]: Check if it is used anywhere else

Comment (hwupathum) [reply apps/console/src/features/applications/constants/application-management.ts:427]: Only used in this repo, and all the occurences are removed with this PR

**Review by brionmario — APPROVED** [review]

**Pull request merged**